Add aud (audience) to JWT properties (#15666)
crate/crate: docs/admin/auth/methods.rst

USER`` with ``JWT`` properties that match those of the provided ``JWT`` token.

Token must contain the following claims:

``kid`` - `Key ID`_.

``iss`` - URL of the `JWK endpoint`_.

``aud`` - `aud`_.

``username`` - user name in a third party app.

``iss`` and  ``username`` values must match the values created by

> ``CREATE USER`` statement. If ``aud`` is provided in ``CREATE USER`` statement,
> token's ``aud`` value must match value created by ``CREATE USER`` statement.
> Otherwise, token's ``aud`` value must match cluster id.

Reviewer: ```suggestion
``CREATE USER`` statement. If ``aud`` has not been defined on the ``CREATE USER`` statement,
the cluster id is used and must match the token's ``aud`` value.
```
